lib/monero/cw_monero.dart

```diff
@@ -61,7 +61,13 @@ class CWMoneroSubaddressList extends MoneroSubaddressList {
   ObservableList<Subaddress> get subaddresses {
     final moneroWallet = _wallet as MoneroWallet;
     final subAddresses = moneroWallet.walletAddresses.subaddressList.subaddresses
-        .map((sub) => Subaddress(id: sub.id, address: sub.address, label: sub.label))
+        .map((sub) => Subaddress(
+          id: sub.id,
+          address: sub.address,
+          label: sub.label,
+          received: sub.balance??"unknown",
```

Comment:
The UI accepts string, and it is already formatted before passing - otherwise we would have to calculate the string on the UI insteaed of the wallet code

Reply:
received is a string, to account for formating so this is expected.